nilearn/decoding/space_net.py

```diff
@@ -34,8 +31,9 @@
 from sklearn.metrics import accuracy_score
 from ..input_data.masker_validation import check_embedded_nifti_masker
 from .._utils.param_validation import _adjust_screening_percentile
-from .._utils.fixes import check_X_y
-from .._utils.fixes import check_cv, center_data
+from sklearn.utils import check_X_y
```

Comment:
Same here: I would move imports to above where cluster of sklearn functions or classed are imported.